src/pages/index.tsx

```diff
@@ -60,6 +60,10 @@ export default function Home({ initialState }: Props) {
     SettingsSelectors.selectEnabledFeatures,
   );
 
+  const isLoggedin =
+    authDisabled &&
+    (session.status !== 'authenticated' || !isClientSessionValid(session));
```

Comment:
incorrect name of variable - we check `shouldLogin` here(see on previous line 118)
```suggestion
  const shouldLogin =
    !authDisabled &&
    (session.status !== 'authenticated' || !isClientSessionValid(session));
```

Reply:
Changed.
Thank you!